README.md

```diff
@@ -27,35 +32,46 @@ roles responsible for building and running apps and operating infrastructure.
 * _Application Operators_ are responsible for configuring the runtime aspects of
   one or more of these microservices. They are the domain experts on the
   platform.
-* _Infrastructure Operators_ are responsible for setting up and maintaining the
+* _Infrastructure Operators_ also known as _Platform Builders_, are responsible for setting up and maintaining the
   infrastructure within which applications run. They are the domain
   experts on the low-level details.
 
 For more details and user stories, see [introduction.md](./introduction.md).
 
 ## See it in action
 
-[Crossplane](https://github.com/crossplane/crossplane) is the implementation of the Open Application Model specification for Kubernetes. To get started with an example of the Open Application Model, follow the guide to run applications on [Crossplane](https://crossplane.io/docs/v0.11/getting-started/run-applications.html).
+[OAM Kubernetes Runtime](https://github.com/crossplane/oam-kubernetes-runtime) is the officially maintained OAM plugin for Kubernetes. 
+
+OAM Kubernetes Runtime is a [joint effort](https://cloudblogs.microsoft.com/opensource/2020/05/27/open-application-model-oam-v1alpha2-crossplane/) with [the Crossplane community](https://github.com/crossplane/crossplane). Furthermore, to get started with an example of using OAM to deliver both applications and cloud resources in unified approach, please follow the end-to-end [getting started doc in Crossplane](https://crossplane.io/docs/v0.11/getting-started/run-applications.html).
```

Comment:
How about we using the latest version `v0.12` for end-to-end documentation?

Reply:
NP